xformers/factory/block_configs.py

```diff
@@ -190,6 +195,7 @@ def __init__(
         multi_head_config_cross: Dict[str, Any],
         position_encoding_config: Optional[Dict[str, Any]] = None,
         layer_norm_style: str = "post",
```

Comment:
Might the distinction between the two variable names be confusing? `layer_norm_style` and `normalization`

Reply:
Sorry missed this comment, that sounds good!